qa/tasks/cephfs/test_mds_metrics.py

```diff
@@ -497,9 +501,14 @@ def test_client_metrics_and_metadata(self):
         self.mount_b.path_to_ino("test.bin")
         self.mount_b.create_files()
 
+        fscid_list = [fs_a.id, fs_b.id]
+        
+        # wait a bit for mgr to get updated metrics
+        time.sleep(5)
+
         # validate
         valid, metrics = self._get_metrics(
-            self.verify_mds_metrics(client_count=TestMDSMetrics.CLIENTS_REQUIRED), 30)
+            self.verify_mds_metrics(client_count=1, mul_fs=fscid_list), 30)
```

Comment:
Can you explain what problem are you fixing here in `test_client_metrics_and_metadata`?

Reply:
The test was failing for multiple filesystems which is done in that test, cause earlier verify metrics was written for only one `cephfs` filesystem.

Reply:
Looks like this is an unrelated fix. Can you move this to a new PR?

Reply:
I see, then this change shouldn't be here. It's just your local branch re-base issue I understand.